wire-ios-request-strategy/Sources/Request Strategies/Conversation/ConversationRequestStrategy.swift

```diff
@@ -47,6 +47,8 @@ public class ConversationRequestStrategy: AbstractRequestStrategy, ZMRequestGene
     let addParticipantActionHandler: AddParticipantActionHandler
     let removeParticipantActionHandler: RemoveParticipantActionHandler
     let updateAccessRolesActionHandler: UpdateAccessRolesActionHandler
+    let createGuestLinkActionHandler: CreateConversationGuestLinkActionHandler
+    let setAllowGuestsAndServicesActionHandler: SetAllowGuestsAndServicesActionHandler
```

Comment:
suggestion: if there's no need to retain them, we can just initialize them directly in the entity sync init below, like other action handlers.